etc/openqa/openqa.ini

```diff
@@ -296,3 +296,14 @@ concurrent = 0
 # lookup_depth = 10
 # Specify at how many state changes the search will be aborted (state = combination of failed/softfailed/skipped modules):
 # state_changes_limit = 3
+
+# Configuration for the OBS rsync plugin
+[obs_rsync]
+project_status_url="$obs_instance/public/build/%%PROJECT/_result";
+concurrency=2
+queue_limit=200
+retry_interval=60
+retry_max_count=2
+home=
+username=openqa-user
+ssh_key_file=~/.ssh/id_rsa
```

Comment:
We normally put a space before and after the `=`-sign so it would be great to do the same here as well for consistency. The quotes around the first entry and the semicolon at the end look wrong as this is plain INI after all.

Reply:
Addressed